pennylane/ops/qubit/matrix_ops.py

```diff
@@ -61,9 +62,9 @@ def __init__(self, *params, wires, do_queue=True):
 
             # Check for unitarity; due to variable precision across the different ML frameworks,
             # here we issue a warning to check the operation, instead of raising an error outright.
-            if not qml.math.allclose(
-                qml.math.dot(U, qml.math.T(qml.math.conj(U))),
-                qml.math.eye(qml.math.shape(U)[0]),
+            if not qmlmath.allclose(
```

Comment:
Why has this change been made?

Reply:
I thought I might as well import `qml.math` as it's own module since we use it over and over. But we can keep using `qml.math` as well.  Both work.